app/views/users/index.html.erb

```diff
@@ -1,5 +1,8 @@
-<% provide :title, @user.name %>
-<h1 style="text-align:left">
-<%= gravatar_for @user %>
-<%= @user.name %>
-</h1>
+<% provide(:title, t("users.all")) %>
+  <h1>
+    <% t("users.all") %>
+  </h1>
+  <ul class="users">
+    <%= render @users %>
+  </ul>
+<%== pagy_bootstrap_nav(@pagy) %>
```

Comment:
có phải dư dấu `=` ko em

Reply:
<%== > bằng với raw => nó sẽ không escape mấy cái giá trị mà không an toàn ở đây @pagy có giá tri tương tự.
 @page=1,
 @pages=5,
 @prev=nil,
 @to=2,
 @vars=
  {:page=>1,
   :items=>2,   
   :outset=>0,
   :size=>[1, 4, 4, 1],
   :page_param=>:page,
   :params=>{},
   :anchor=>"",
   :link_extra=>"",
   :item_path=>"pagy.info.item_name",
   :count=>10}>